internal/platform/types.go

```diff
@@ -62,6 +63,34 @@ func (info OpenShiftVersion) String() string {
 		"Version: " + info.Version + "]"
 }
 
+func (v OpenShiftVersion) Compare(o OpenShiftVersion) (int, error) {
+	if d, err := compareSegment(v.MajorVersion(), o.MajorVersion()); d != 0 {
+		return d, err
+	}
+	if d, err := compareSegment(v.MinorVersion(), o.MinorVersion()); d != 0 {
+		return d, err
+	}
+	return 0, nil
+}
+
+func compareSegment(v, o string) (int, error) {
+	v1, err := strconv.Atoi(v)
```

Comment:
Doesn't this mean 4.3.11 would be greater than 4.4.1?

Reply:
How so? Major is compared and only if equal, then minor is compared.

Reply:
I see "4.3.11" < "4.4.1" from the code.